modules/home-manager/default.nix

```diff
@@ -1,4 +1,23 @@
-{ lib, pkgs, ... }@args:
+{ lib
+, config
+, options
+, ...
+}@args:
+
+let
+  cfg = config.programs.catppuccin-global;
+in
 {
   imports = import ../lib/mkImports.nix args ./.;
+
+  options.programs.catppuccin = lib.ctp.mkCatppuccinOpt "catppuccin global";
+
+  config.programs = lib.mkIf cfg.enable (
+    lib.foldlAttrs
```

Comment:
I wish I could have used `builtins.mapAttrs`, but I couldn't get it done with it, I don't know why yet. Do you have a clue?